sashimi-webapp/src/database/retrieve/query.js

```diff
@@ -102,8 +102,8 @@ export default class query {
     return new Promise((resolve, reject) =>
       sqlCommands.loadFile(fileId)
       .then((fileContent) => {
-        fileContent = stringManipulator.replaceAll(fileContent, '\\"', '"');
-        resolve(fileContent);
+        const resolvedFileContent = stringManipulator.replaceAll(fileContent, '\\\\"', '"');
```

Comment:
I just read somewhere it should be something to do with the Javascript parser, which `'\\'` will be ignored and to search for `'\'` need four backslash